blog/2015/11/intel-sponsor-wise-uk.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: blog
+root: ../../..
+author: Aleksandra Pawlik
+title: "Intel to sponsor the first Workshop for Women in Science and Engineering in the UK"
+date: 2015-11-25
+time: "20:00:00"
+category: ["WiSE", "Women in Science and Engineering"]
+---
+<!-- start excerpt -->
+<p>We are very happy to announce that the first <a href="http://apawlik.github.io/2015-12-wise-uk/">
+UK Software Carpentry Workshop for Women in Science and Engineering</a> 
+received generous support from <a href="http://www.intel.co.uk/">Intel</a>. 
+The event will take place on 14th and 15th December at the University of Manchester.
+</p>
+<!-- end excerpt -->
+<div style="text-align: center">
+<a href="http://www.intel.co.uk/">
+<img src="http://apawlik.github.io/2015-12-wise-uk/img/intel-logo.jpg" alt="Intel logo" style="height:120px"/></a>
```

Comment:
Please copy the image into the blog/2015/11 directory and refer to the local copy.